src/site/content/en/blog/command-click-to-add-multiple-cursors-in-the-sources-panel/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
```

Comment:
This is DevTools, if the sheet is saying to put it on this site that's a mistake - sorry. This would need to go on dcc.

Reply:
No worries. I've edited the sheet and I'll add this to the next batch I'll do for DCC.